Assets/Scripts/Menu.cs

```diff
@@ -33,6 +37,26 @@ public void LoadButton_OnClick()
       loadMenu.SetActive(true);
     }
 
+    public void PlayerMovement_OnClick()
+    {
+      PlayerMovementMode playerMovementMode = SwitchPlayerMovement();
+      SetPlayerMovementText(playerMovementMode);
+    }
+
+    private void SetPlayerMovementText(object value)
+    {
+      playerMovementText.text = $"{c_playerMovement}: {System.Enum.GetName(typeof(PlayerMovementMode), value)}";
+    }
+
+    private PlayerMovementMode SwitchPlayerMovement()
```

Comment:
Consider changing name to GetNextPlayerMovementMode. This method does not actually switch anything.

Reply:
Right, it soes in the PlayerPrefs. It would be better if it switched a private field in this class as well and not return anything. Methods that change state AND return values should be avoided when possible.